Add `Gameplay/Game modifier/Mod multiplier (lazer)` (#15136)
ppy/osu-wiki: wiki/Gameplay/Game_modifier/Mod_multiplier_(lazer)/en.md

---
stub: true
tags:
  - score multiplier
  - score
  - mod
  - mod multiplier
---

# Mod multiplier (lazer)

::: alert-note
**Note:** For the osu!(stable) version of this article, see [Mod multiplier](/wiki/Gameplay/Game_modifier/Mod_multiplier)
:::

::: alert-note
**Note:** Not to be confused with [Combo](/wiki/Gameplay/Combo_(score_multiplier))
:::

## Description

The **mod multiplier** is an attribute of each [game modifier](/wiki/Gameplay/Game_modifier_(lazer)) that affects the value of a player's score on a [beatmap](/wiki/Beatmap).

> By default, the mod mutiplier is `1.00x`, but some mods can reduce or increase[^mania-note] it. Typically, mods from the `Difficulty Reduction` category have multipliers less than `1.00x`, while mods from the `Difficulty Increase` category have multipliers greater than `1.00x` to reward plays of increaed difficulty with a higher score.

Reviewer: ```suggestion
By default, the mod mutiplier is `1.00x`, but some mods can reduce or increase[^mania-note] it. Typically, mods from the `Difficulty Reduction` category have multipliers less than `1.00x`, while mods from the `Difficulty Increase` category have multipliers greater than `1.00x` to reward plays of increased difficulty with a higher score.
```

The multiplier of a mod can also depend on its customisation settings. Detailed explanations on how the mod multipliers are calculated can be found in the [articles of the respective mods](/wiki/Gameplay/Game_modifier_(lazer)#list-of-mods) or in osu!(lazer)'s source code:

- [`OsuScoreMultiplierCalculatorV2`](https://github.com/ppy/osu/blob/d9c73e12adff2feaae4a3e158d36fe5883faf6ca/osu.Game.Rulesets.Osu/Scoring/OsuScoreMultiplierCalculatorV2.cs) in osu!(lazer)'s source code
- [`TaikoScoreMultiplierCalculator`](https://github.com/ppy/osu/blob/d9c73e12adff2feaae4a3e158d36fe5883faf6ca/osu.Game.Rulesets.Taiko/Scoring/TaikoScoreMultiplierCalculator.cs) in osu!(lazer)'s source code
- [`CatchScoreMultiplierCalculator`](https://github.com/ppy/osu/blob/d9c73e12adff2feaae4a3e158d36fe5883faf6ca/osu.Game.Rulesets.Catch/Scoring/CatchScoreMultiplierCalculator.cs) in osu!(lazer)'s source code

> - [`ManiaScoreMultiplierCalculator`](https://github.com/ppy/osu/blob/d9c73e12adff2feaae4a3e158d36fe5883faf6ca/osu.Game.Rulesets.Mania/Scoring/ManiaScoreMultiplierCalculator.cs) in osu!(lazer)'s source code

Reviewer: as I was finishing the review, I noticed these links repeat somewhere down below. in these cases, it is better to create a named reference style link and reuse it when needed: https://osu.ppy.sh/wiki/ascf#links

Author: Applied in https://github.com/ppy/osu-wiki/pull/15136/changes/c82a9c9a66c3fc509239a49245bde84191e4c79b (and https://github.com/ppy/osu-wiki/pull/15136/commits/278d4563c20d071afa16341c01471419247a29a6, oops), although only the first link is used more than once.

## History

> Initially, osu!(lazer) used the same constant mod multipliers as osu!(stable) (with the exception of [Key mods](/wiki/Gameplay/Game_modifier/Key_mods_(lazer)), see [below](#reduction-of-osu!mania-key-mods-multiplier)),[^mods-2017] which can be found [here](/wiki/Gameplay/Game_modifier/Summary). However, due to a different scoring system and new features like mod customisation, these multipliers were not suitable for the game in the long run. Therefore, the system was adjusted multiple times in the following years.

Reviewer: ```suggestion
Initially, osu!(lazer) used the same constant mod multipliers as osu!(stable) (with the exception of [Key mods](/wiki/Gameplay/Game_modifier/Key_mods_(lazer)), see [below](#reduction-of-osu!mania-key-mods-multiplier)),[^mods-2017] which can be found in [the mod overview article](/wiki/Gameplay/Game_modifier/Summary). However, due to a different scoring system and new features like mod customisation, these multipliers were not suitable for the game in the long run. Therefore, the system was adjusted multiple times in the following years.
```

Reviewer: also "different scoring system" should probably lead to the score v2 part of the score article

Author: Hm I guess, although technically lazer doesn't use ScoreV2.

I think linking to https://osu.ppy.sh/wiki/en/Help_centre/Upgrading_to_lazer#differences-in-scoring might make more sense

Reviewer: oh yeah

### Introduction of progressive multipliers for rate adjust mods

[On 19 April 2023](https://osu.ppy.sh/home/changelog/lazer/2023.419.0), the score multipliers for some previously adjusted mods were changed again.[^linear-rate-multipliers] Namely, [Double Time (DT)](/wiki/Gameplay/Game_modifier/Double_Time_(lazer)) and [Nightcore (NC)](/wiki/Gameplay/Game_modifier/Nightcore_(lazer)) multipliers now scaled linearly from `1.00x` to `1.20x` in 0.1 steps, while [Half Time (HT)](/wiki/Gameplay/Game_modifier/Half_Time_(lazer)) and [Daycore (DC)](/wiki/Gameplay/Game_modifier/Daycore) multipliers scaled linearly from `0.50x` to `0.90x` in 0.1 steps. In an update [on 18 December 2023](https://osu.ppy.sh/home/changelog/lazer/2023.1218.0), these were further adjusted to range from `0.10x` to `0.50x` instead.[^ht-dc-multiplier]

> Specifically, the mod multipliers for rate adjust mods that were in use until the [mod multiplier rebalance](#mod-multiplier-rebalance) in 2026 are summarised in the following table:

Reviewer: dunno but maybe this reads a bit easier
```suggestion
Specifically, the mod multipliers for rate adjust mods prior to the [multiplier rebalance](#mod-multiplier-rebalance) in 2026 are summarised in the following table:
```

| Speed adjust | ![][osu!] ![][osu!taiko] ![][osu!catch] | ![][osu!mania] |
| :-- | :-- | :-- |
| 0.50x - 0.59x | `0.10x` | `0.10x` |
| 0.60x - 0.69x | `0.20x` | `0.20x` |
| 0.70x - 0.79x | `0.30x` | `0.30x` |
| 0.80x - 0.89x | `0.40x` | `0.40x` |
| 0.90x - 0.99x | `0.50x` | `0.50x` |
| 1.01x - 1.09x | `1.00x` | `1.00x` |
| 1.10x - 1.19x | `1.02x` | `1.00x` |
| 1.20x - 1.29x | `1.04x` | `1.00x` |
| 1.30x - 1.39x | `1.06x` | `1.00x` |
| 1.40x - 1.49x | `1.08x` | `1.00x` |
| 1.50x - 1.59x | `1.10x` | `1.00x` |
| 1.60x - 1.69x | `1.12x` | `1.00x` |
| 1.70x - 1.79x | `1.14x` | `1.00x` |
| 1.80x - 1.89x | `1.16x` | `1.00x` |
| 1.90x - 1.99x | `1.18x` | `1.00x` |
| 2.00x | `1.20x` | `1.00x` |

### Reduction of osu!mania key mods multiplier

> In osu!(stable), [Key mods](/wiki/Gameplay/Game_modifier/Key_mods_(lazer)) use score multipliers smaller than `1.00x`, as listed in [xK](/wiki/Gameplay/Game_modifier/xK). When initially implemented in osu!(lazer), the score multiplier for all of these was set to `1.00x`. This allowed players to reduce the key count on [converted](/wiki/Beatmap/Converts) beatmaps without any score penalty. To avoid this, an update [on 16 August 2025](https://osu.ppy.sh/home/changelog/lazer/2025.816.0), changed all the multipliers to `0.90x`.[^key-mods-multiplier]

Reviewer: ```suggestion
In osu!(stable), [key mods](/wiki/Gameplay/Game_modifier/Key_mods_(lazer)) use score multipliers smaller than `1.00x`, as listed in [xK](/wiki/Gameplay/Game_modifier/xK). When initially implemented in osu!(lazer), the score multiplier for all of these was set to `1.00x`. This allowed players to reduce the key count on [converted](/wiki/Beatmap/Converts) beatmaps without any score penalty. To avoid this, an update [on 16 August 2025](https://osu.ppy.sh/home/changelog/lazer/2025.816.0) changed all the multipliers to `0.90x`.[^key-mods-multiplier]
```

### Mod multiplier rebalance

> As mentioned above, the score multipliers initially used in osu!(lazer) did not suit the new game client and were generally not well received by the community. For example, scores achieved in osu!(lazer) often appeared higher on global leaderboards than scores from osu!(stable), despite having worse combo and accuracy. This was due to the [Classic (CL)](/wiki/Gameplay/Game_modifier/Classic) mod, which is automatically applied to all osu!(stable) scores, only having a multiplier of `0.96x` at the time.

Reviewer: > and were generally not well received by the community

while I can agree to this (as I think I have occasionally seen the outbursts), adding a reference to these complaints or two would be helpful (especially if they are met with unanimous support)

Author: I though of this too but didn't know what to link to. I'll look for some references tomorrow.

Author: https://github.com/ppy/osu-wiki/pull/15136/changes/68a8a21c30db538506540184d0a0edf0aff1a68e
